python-spec/src/somacore/ephemeral/collections.py

```diff
@@ -154,6 +162,32 @@ def transformations(self) -> MutableMapping[str, coordinates.CoordinateTransform
         raise NotImplementedError()
 
 
+class Image2D(  # type: ignore[misc]   # __eq__ false positive
+    BaseCollection[base.SOMAObject], images.Image2D[data.DenseNDArray, base.SOMAObject]
+):
```

Comment:
I created an image collection for 2D images only. This is slightly different from early high-level discussions. I am not adding a 3D image collection now, but we can add one in the future.